include/compose.bash

```diff
@@ -216,6 +216,9 @@ cloudbreak:
         - CB_SMTP_SENDER_FROM=$CLOUDBREAK_SMTP_SENDER_FROM
         - CB_BAYWATCH_ENABLED=$CB_BAYWATCH_ENABLED
         - CB_BAYWATCH_EXTERN_LOCATION=$CB_BAYWATCH_EXTERN_LOCATION
+        - CB_CERT_DIR=/certs
```

Comment:
I have just thought about a bit and discussed with Lajcsi. The java code shall be defaulted to /cert and these values  shall be eliminated from cbd. The java code shall  automatically.

Anyway at the moment when you use it from IDEA it does not work out of the box, since you need some interaction either you need to copy the generated files to /tmp/certs or set the -D parameter variable to point to that directory where the cbd has generated the certs.